### ISSUE TITLE
Handling N in the sequence

### ISSUE THREAD

**Issue by khodor14** [vgteam/vg#3467]
I'm using vg construct to construct a linear graph
here is the command : vg construct -r genomes/AE005174.2.fa > x.vg
here is the warning: warning:[vg::Constructor] Unsupported IUPAC ambiguity codes found in ENA|AE005174|AE005174.2; coercing to N.

Is there a way to handle this?

**Comment by faithokamoto** [comment]
Yeah, Glenn answered with how we "handle" this: by converting to N. It's not an error, just a warning.
